src/modules/services_manager_service.rb

```diff
@@ -72,6 +68,14 @@ def list_units
         SCR.Execute(Path.new('.target.bash_output'), command)
       end
 
+      def is_active? service
```

Comment:
On the other hand, it would be fine if you add documentation to this method as it's a public one.